RptToXml/RptDefinitionWriter.cs

```diff
@@ -566,13 +566,19 @@ private void GetFieldObject(Object fo, ReportDocument report, XmlWriter writer)
 			}
 			else if (fo is FormulaFieldDefinition ff)
 			{
-                writer.WriteStartElement("FormulaFieldDefinition");
+				dynamic rasField = GetRASField(ff);

```

Comment:
why is this typed dynamic?

Reply:
This is not needed. You are right. The proper Type is CRDataDefModel.FormulaField. I will change this